### PR TITLE
js style to object

### PR DESCRIPTION
**Pull request by georgiyevv**

### CODE REVIEW

File: src/convertToObject.js [hunk 1]
```diff
@@ -1,12 +1,70 @@
...
 
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
   // write your code here
+    const separator = ';';
+    const styles =  formatString(sourceString).split(separator);
```
Comment (etojeDenys) [inline src/convertToObject.js:18]:
```suggestion
    const styles = sourceString.split(separator);
```
looks redundant

File: src/convertToObject.js [hunk 2]
```diff
@@ -1,12 +1,70 @@
...
+    const object = {};
+
+    for (const style of styles) {
+      const values = style.split(':');
+      const [property, value] = values;
```
Comment (etojeDenys) [inline src/convertToObject.js:23]:
```suggestion
      const [property, value]  = style.split(':');
```

**Review by etojeDenys — CHANGES_REQUESTED** [review]

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Still not fixed from the previous review

File: src/convertToObject.js [hunk 3]
```diff
@@ -1,12 +1,56 @@
...
+  -ms-user-select: none;
+  user-select: none;
+
+
+  white-space: nowrap;
+  width: auto;
+`;
+
+
+convertToObject(complexStylesString);
```
Comment (VitaliyBondarenko1982) [inline src/convertToObject.js:56]: all tests failed because no need to replace module.exports

```suggestion
  module.exports = convertToObject;
```

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
Solution looks good.
Check comment, please,  and try to fix tests.

**Review by VitaliyBondarenko1982 — APPROVED** [review]
Well done

**Pull request closed without merging**